src/src.pro

```diff
@@ -18,8 +18,8 @@
 #    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             #
 ############################################################################
 
-lessThan(QT_MAJOR_VERSION, 5) {
-  error("requires Qt 5.0 or later")
+lessThan(QT_MAJOR_VERSION, 5)|lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6) {
```

Comment:
:beetle:  Humm, logic test failure with:

    lessThan(QT_MAJOR_VERSION, 5)|lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6) {
        error("Mudlet requires Qt 5.6 or later")
    }

I could get a Qt 4.x kit selection to proceed (with a moan about an unknown `uitools` module),  It seems that OR/AND logic grouping is not as one expects but explicit grouping with parenthesis fixes the issue:

    lessThan(QT_MAJOR_VERSION, 5)|(lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6)){
        error("Mudlet requires Qt 5.6 or later")
    }